checker/tests/index/MinLenFromPositive.java

```diff
@@ -0,0 +1,21 @@
+import org.checkerframework.checker.index.qual.Positive;
+import org.checkerframework.common.value.qual.*;
+
+class MinLenFromPositive {
```

Comment:
Please add more tests.  They should include testing lub and testing when the return type of a method is `@Positive`.